docs/source/models.rst

```diff
@@ -274,14 +350,33 @@ and the output is the batch size and is thus set to -1 to allow for variable bat
   signature:
       inputs: '[{"name": "images", "dtype": "uint8", "shape": [-1, 28, 28, 1]}]'
       outputs: '[{"shape": [-1, 10], "dtype": "float32"}]'
+      params: null
+
+Signature with params Example
+"""""""""""""""""""""""""""""
+The params field is optional and is used to specify parameters that can be used for model inference.
+Params accept scalar values of type :py:class:`MLflow data types <mlflow.types.DataType>`, or a list
+of such values. The default value of a parameter is specified by setting the ``default`` field, and the value
+should be of the type specified by ``type`` field. The ``shape`` field can be used to specify the shape 
+of the value, it should be ``None`` for scalar values and ``(-1,)`` for a list.
+
+.. code-block:: yaml
+
+    signature:
+        inputs: '[{"name": "text", "type": "string"}]'
+        outputs: '[{"name": "output", "type": "string"}]'
+        params: '[{"name": "temperature", "type": "float", "default": 0.5, "shape": null},
+                  {"name": "top_k", "type": "integer", "default": 1, "shape": null},
+                  {"name": "suppress_tokens", "type": "integer", "default": [101, 102], "shape": [-1]}]'
```

Comment:
@serena-ruan Does it mean the "shape" is only supposed to be used with tensor values?
If "shape" also applies to lists, then for a list like `[101, 102]`, shouldn't the "shape" of it be `(2,)`?
Maybe this can be clarified by providing an example of a value whose shape is `(2, 3)`, so that people understand the regular case.

Reply:
`shape` is supposed to only be used for list/array values, and currently `params` only support scalar value or 1D array value, so `shape` should either be `None` or `(-1,)`. And for list like in this case `[101, 102]` it's a 1-dimensional array so the shape is (-1,), actually for whatever 1D array shape (-1,) always work.